sagepay-server-util.js

```diff
@@ -4,10 +4,9 @@ const assert = require("assert");
 const bodyParser = require("body-parser");
 const crypto = require("crypto");
 const extend = require("extend");
-const querystring = require("querystring");
 const request = require("request-promise-native");
 
-const parser = bodyParser.urlencoded();
+const parser = bodyParser.urlencoded({extended: false});
```

Comment:
Consider if using default configuration creates any harm in the context of its use.

Reply:
Using the default configuration displays a warning on the console that the default will change in the next major version update.

https://github.com/expressjs/body-parser#extended

I'd rather avoid displaying the warning, and I think that `extended: false` is better (safer). We could use true.